packages/user-profile-spa/src/app/app.component.ts

```diff
@@ -8,7 +8,46 @@ import { Component, OnInit } from '@angular/core';
 
 export class AppComponent implements OnInit{
   userDetails: UserType;
+  title = 'User Profile';
+  footerLinks = [
+    {
+      category: 'Quick Links',
+      links: [
+        { text: 'One Platform in Mojo', href: 'https://mojo.redhat.com/groups/pnt-devops/projects/one-portal/' },
+        { text: 'Weekly Blog', href: 'https://mojo.redhat.com/groups/pnt-devops/projects/one-portal/blog/' },
+        { text: 'Contact Us', href: '/contact-us' },
+      ]
+    },
+    {
+      category: 'Related sites',
+      links: [
+        { text: 'access.redhat.com', href: 'https://access.redhat.com/' },
+        { text: 'catalog.redhat.com', href: 'https://catalog.redhat.com/' },
+        { text: 'connect.redhat.com', href: 'https://connect.redhat.com/' },
+      ]
+    },
+    {
+      category: 'Help',
+      links: [
+        { text: 'Report An Issue' },
+        { text: 'One Portal FAQs', href: 'https://mojo.redhat.com/docs/DOC-1225598' },
+        { text: 'one-portal@redhat.com', href: 'mailto:one-portal@redhat.com' },
+        { text: 'GitHub', href: 'https://github.com/1-platform/one-platform' }
+      ]
+    },
+  ];
+  breadcrumbs = [
+    { name: 'Home', href: '/' },
+    { name: 'User Profile', href: '/user' },
+  ];
 
```

Comment:
Same suggestion as above, would be great to set up a conf file for storing these values. Maybe re-use the same footer conf across multiple packages.

Reply:
Setup a json which saves those values